src/dispatch/incident/flows.py

```diff
@@ -351,41 +477,40 @@ def incident_create_flow(*, incident_id: int, checkpoint: str = None, db_session
     incident = incident_service.get(db_session=db_session, incident_id=incident_id)
 
     # get the incident participants based on incident type and priority
-    individual_participants, team_participants = get_incident_participants(
-        db_session, incident.incident_type, incident.incident_priority, incident.description
-    )
+    individual_participants, team_participants = get_incident_participants(incident, db_session)
 
     # add individuals to incident
     for individual in individual_participants:
         participant_flows.add_participant(
-            db_session=db_session, user_email=individual.email, incident_id=incident.id
+            user_email=individual.email, incident_id=incident.id, db_session=db_session
         )
 
-    log.debug(f"Added {len(individual_participants)} participants to incident id {incident.id}.")
-
-    # create the incident ticket
-    ticket = create_incident_ticket(
-        incident.title,
-        incident.incident_type.name,
-        incident.incident_priority.name,
-        incident.commander.email,
-        incident.reporter.email,
-        incident.visibility,
+    event_service.log(
+        db_session=db_session,
+        source="Dispatch App",
+        description="Incident participants added to incident",
```

Comment:
Is this an event? Wouldn't we more likely to know when person X was added?

Reply:
We do it in the add participant flow.